Tasks/AzureRmWebAppDeployment/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -1,7 +1,7 @@
 {
   "loc.friendlyName": "Azure App Service Deploy",
   "loc.helpMarkDown": "[More Information](https://aka.ms/azurermwebdeployreadme)",
-  "loc.description": "Update Azure App Services, Function Apps, Mobile Apps using Web Deploy / Kudu REST APIs",
+  "loc.description": "Update Azure Web App Services, Web App On Linux , Function Apps, Mobile Apps using Web Deploy / Kudu REST APIs",
```

Comment:
Long back we changed `AzureRm Web App` to `Azure App Service`
why we are renaming it again ?